src/containerapp/setup.py

```diff
@@ -37,7 +37,8 @@
 
 # TODO: Add any additional SDK dependencies here
 DEPENDENCIES = [
-    'azure-cli-core'
+    'azure-cli-core',
+    'pycomposefile>=0.0.29'
 ]
 
```

Comment:
We'll also need an entry in the history.rst file for this change

Reply:
Also, it looks like some CI steps are failing. We'll need to fix some tests / test recordings to get those passing

Reply:
HISTORY.rst file has been updated and CI tests have been re-recorded.